src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1907,6 +1920,110 @@ void NotationActionController::toggleStrike()
     currentNotationInteraction()->toggleStrike();
 }
 
+bool NotationActionController::isMasterNotation(const INotationPtr notation) const
+{
+    return globalContext()->currentMasterNotation()->notation() == notation;
```

Comment:
You need to check everywhere that the current notation is not null, otherwise if an action is sent when the notation is not yet open, a crash will occur